jsonrpc/types.go

```diff
@@ -34,6 +34,7 @@ type transaction struct {
 	BlockNumber *argUint64     `json:"blockNumber"`
 	TxIndex     *argUint64     `json:"transactionIndex"`
 	Type        argUint64      `json:"type"`
+	ChainId     *argUint64     `json:"chainId"`
```

Comment:
This file is also missing a change to [toTransaction](https://github.com/0xPolygon/polygon-edge/blob/develop/jsonrpc/types.go#L54) to include the `chainId` in the JSON response.

Presently, trying to make the change is tricky because it requires a mapping from `*big.Int` to `*argUint64`.

Perhaps the type here should be changed to `argBig` so the conversion can become: `argBig(*t.ChainID)`

Reply:
Im going to change the scope of this PR to only be focused on the renaming vs the addition of the other vars. the addition of ChainId will come with the other txn hash fixes.